### ISSUE TITLE
DateTimeInput not rendered with 'contained' variant as is default for other controls

### ISSUE THREAD

**Issue by qpointsystems** [marmelab/react-admin#3892]
**What you were expecting:**
I'd expect the DateTimeInput control on a form to render with the newly favored variant flavor of 'contained'.

**What happened instead:**
 It seems to render in the pre-Beta default theme

*** Including screen shot

```
<FormTab label="bh.labels.tabs.basics" icon={<HomeIcon />} disableRipple={true}>    
  <ReferenceInput source="class_id" label="bh.labels.parentClass" reference="classes" disabled>
     <SelectInput optionText="name" optionValue="id"/>
  </ReferenceInput>    
  <NameInput source="name" label="bh.labels.name" autoFocus/>
  <LabelInput source="label" label="bh.labels.label"/>
  <FormGroupRow maxWidth={500}>
    <DateTimeInput source="begin_date" label="Begin Date"/>
    <DateTimeInput source="end_date" label="End Date"/>
  </FormGroupRow>
   <TagsField /> 
</FormTab>     
```

**Other information:**
Screenshot
![DateFields](https://user-images.githubusercontent.com/44985224/67704067-ad8f9680-f971-11e9-87cd-076920fed855.png)

**Environment**

* React-admin version: Beta 3.0.0
* Last version that did not exhibit the issue (if applicable): Pre-Beta
* React version: ^16.9.0
*** Material UI/core version: ^4.3.3**
* Browser: Version 77.0.3865.90 (Official Build) (64-bit)
* Stack trace: N/A

**Comment by fzaninotto** [comment]
Reproduced and confirmed, thanks.

**Comment by fzaninotto** [comment]
fixed by #3897
